src/m365/spo/commands/folder/FolderColor.ts

```diff
@@ -0,0 +1,19 @@
+export const FolderColorValues: { [key: string]: string } =
+{
+  yellow: '0',
+  darkRed: '1',
+  darkOrange: '2',
+  darkGreen: '3',
+  darkTeal: '4',
+  darkBlue: '5',
+  darkPurple: '6',
+  darkPink: '7',
+  darkGrey: '8',
```

Comment:
Seems like a made a little mistake here. Instead of `darkGrey`, the color should be just `grey`.